posthog-core/src/index.ts

```diff
@@ -720,33 +722,18 @@ export abstract class PostHogCoreStateless {
    *** QUEUEING AND FLUSHING
    ***/
   protected enqueue(type: string, _message: any, options?: PostHogCaptureOptions): void {
```

Comment:
could allow a callback here
then convert that to a promise in some interfaces and it's not breaking for others?

(have not thought this through though!)

Reply:
silly question though

if we set `maxBatchSize` to 1 is that (almost) the same as this?

Reply:
yeah since it does not return a promise to be awaited, its the same as `maxBatchSize=1`